Dockerfile

```diff
@@ -88,13 +88,13 @@ RUN . /opt/spack/share/spack/setup-env.sh && spack load cuda && \
     cmake -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_COMPILER=g++ -DENABLE_CUDA=On -DCMAKE_CUDA_STANDARD=14 -DCMAKE_CUDA_ARCHITECTURES=70 -DENABLE_OPENMP=On .. && \
     make -j 4
 
-FROM ghcr.io/rse-ops/hip-ubuntu-20.04:hip-4.3.1 AS hip
+FROM ghcr.io/rse-ops/hip-ubuntu-20.04:hip-5.1.3 AS hip
```

Comment:
Do we want to stick with 5.1 over 5.2 or 5.3 for this release?

Reply:
5.1.3 is the newest image in the rse-ops repo